lua/autorun/server/sv_spawn_protection.lua

```diff
@@ -136,7 +136,7 @@ local function weaponIsAllowed( weapon )
 end
 
 local function keyVoidsSpawnProtection( keyCode )
```

Comment:
Now that the table is named `keyVoidsSpawnProtection`, you need to remove this function and use the table instead.